### DISCUSSION TITLE
New Home & Continuing Maintenance

### DISCUSSION THREAD

**Discussion by navarr** [wardenenv/warden#556] [Announcements]
Hello everyone!

The Warden Project is pleased to announce the addition of @navarr as a maintainer, in addition to it's new home at https://github.com/wardenenv.  Despite this move, it is expected that your day to day operations, if you are using Warden, will remain the same at current.

We are still working on the process of migrating everything out of @davidalger's individual responsibility to group responsibility, and we're eager to continue maintenance with a renewed pace in 2023.  Related to that, we have started an [Open Collective for Warden](https://opencollective.com/warden) which is being administered by @navarr.  The current goal of this is to crowd-fund the domain name - which the .dev registry views as a premium domain, as hosting continues to be free.

## Roadmap

Our roadmap at current is to bring in the changes made in the [Den fork](https://github.com/swiftotter/den) in a way that is as backwards-compatible as possible and does not affect existing installs.

That includes:

* Documentation and Features to make the switch from popular forks back to Warden proper
* Complete ARM64 support through the use of Alpine Linux as an opt-in for existing projects and default for newly initialized projects
* Backporting Enhancements and new available services versions as listed in [the Den changelog](https://swiftotter.github.io/den/changelog.html)
* Combing through Issues and PRs as part of continuing maintenance

You can track progress of the major roadmap updates on issue #555 

Thank you for your patience and support!

**Comment by davidalger** [comment]
Thank you @navarr for putting this together and spearheading the initiative for a more open and collaborative future. I'm excited to see a better and brighter year for Warden come 2023! 🥂

**Reply by navarr** [reply]
I've loved Warden since the day you introduced me to it!

Thank you so much for starting it and taking care of it over the years.  Fingers crossed 🤞  that we'll ease as much responsibility as possible with the new initiative.

**Reply by dudzio12** [reply]
I've started to lose faith, so it's great to hear it guys! Good luck @navarr 🚀

**Reply by prestonchoate** [reply]
This is amazing news! Looking forward to the future of this project and contributing where I can

**Reply by gwillem** [reply]
Congratulations! Would you prefer GitHub or opencollective donations?

**Reply by navarr** [reply]
@gwillem I know I already answered you in private, but I'll note it here, too - we accept funding through GitHub or OpenCollective - whichever is most convenient to you.  At this time, we do not have a preference.  We're appreciative of each and every dollar.  All of which will be accounted for openly at https://opencollective.com/warden

Thank you for your Sponsorship!

**Reply by adam-paterson** [reply]
Excellent news!

**Comment by CRC-Mismatch** [comment]
Those are excellent news! All praises to you @navarr , you're doing everything with excellence!
On a side note, I noticed that the change from `docker-compose` to `docker compose` was sidelined and excluded from this release. Is there an ETA on when we should expect to see it in a release? I'm asking just to have a rough estimate, because until then I have to remain vigilant with new coworkers, or those who switch machines (or reformat their current ones), since for now they'll need to install docker-compose standalone...

**Reply by navarr** [reply]
No timeline on this yet
